test/metabase/query_processor/dashboard_test.clj

```diff
@@ -196,7 +202,15 @@
                  (mt/rows (run-query-for-dashcard
                            dashboard-id card-id dashcard-id
                            :parameters [{:id    "5791ff38"
-                                         :value ["Something Else"]}])))))))))
+                                         :value ["Something Else"]}])))))
+        (testing "Providing a nil value should override both defaults and produce an error"
```

Comment:
```suggestion
        (testing "Specifying a nil value for a required parameter will throw an error"
```

I'm not sure I follow the meaning of the original test string, I think this is more appropriate